rust_src/remacs-lib/math.rs

```diff
@@ -0,0 +1,22 @@
+extern crate libc;
```

Comment:
I think this should be on the crate root and import it in submodules with `use libc;`